core/common/src/ElementProps.ts

```diff
@@ -12,7 +12,16 @@ import {
   Id64String,
   Logger,
 } from "@bentley/bentleyjs-core";
-import { AngleProps, LowAndHighXY, LowAndHighXYZ, TransformProps, XYProps, XYZProps, YawPitchRollProps } from "@bentley/geometry-core";
+import {
+  AngleProps,
+  ClipVectorProps,
+  LowAndHighXY,
+  LowAndHighXYZ,
+  TransformProps,
+  XYProps,
+  XYZProps,
+  YawPitchRollProps,
+} from "@bentley/geometry-core";

```

Comment:
Fix import formatting?

Reply:
We need to start using prettier instead of wasting time on silly stuff.